test/test_datapipe.py

```diff
@@ -1149,20 +1149,30 @@ def _helper(ref_fn, fn, input_col=None, output_col=None):
             for constr in (list, tuple):
                 datapipe = dp.iter.IterableWrapper([constr((0, 1, 2)), constr((3, 4, 5)), constr((6, 7, 8))])
                 res_dp = datapipe.map(fn, input_col, output_col)
-                ref_dp = datapipe.map(ref_fn)
+                ref_dp = datapipe.map(ref_fn) if ref_fn is not None else datapipe
                 self.assertEqual(list(res_dp), list(ref_dp))
                 # Reset
                 self.assertEqual(list(res_dp), list(ref_dp))
 
+        p_fn_n1 = partial(fn_n1, d1=1)
+
+
         # Replacing with one input column and default output column
         _helper(lambda data: (data[0], -data[1], data[2]), fn_11, 1)
         _helper(lambda data: (data[0], (-data[1], data[1]), data[2]), fn_1n, 1)
+
+        _helper(lambda data: (data[0], data[1], data[0] + data[1]),
+                lambda d0, d1: d0 + d1, (0, 1), 2)
+        _helper(lambda data: (data[0], data[1], 1 + data[1]), p_fn_n1, 1, 2)
```

Comment:
Could you please add two tests for the function with the default value?
- One test is emitting the parameter with the default value
- One test is overriding the parameter with the default value

Example
```py
def fn(a, b=1):
    return a + b

ref_dp = datapipe.map(fn, input_col=0)  # using default value
ref_dp = datapipe.map(fn, input_col=(0, 1))  # overriding default value

```

Reply:
Done, https://github.com/pytorch/pytorch/pull/79344/commits/3ca662848d5ab34f9c99f00d31c22b5f4f968ca2